src/show/pages/private/CreateProject/index.js

```diff
@@ -27,9 +29,11 @@ const CreateProject = () => {
     deadline: '',
     images: ['', '', '', '', '', '', '', ''],
     team_members: [{}, {}, {}, {}, {}, {}, {}],
-    user_id: 1
+    user_id: currentUser.id
```

Comment:
You can get rid of this line by going into the projects controller and in the create method you can `assign_attributes(user_id: current_user.id)` to the `project_params`.